lib/activerecord/delay_touching.rb

```diff
@@ -6,9 +6,9 @@ module DelayTouching
     extend ActiveSupport::Concern
 
     # Override ActiveRecord::Base#touch.
-    def touch(name = nil)
+    def touch(*names)
       if self.class.delay_touching? && !try(:no_touching?)
-        DelayTouching.add_record(self, name)
+        DelayTouching.add_record(self, names)
```

Comment:
I would argue that this is "shorter" but not necessarily "cleaner". The benefit of the separate method is that it's more explicit. I don't feel strongly here, though. Perhaps we need a 3rd opinion to sway the result?